synchronizer/l1_sync_orchestration.go

```diff
@@ -58,20 +60,15 @@ func newL1SyncOrchestration(ctx context.Context, producer l1RollupProducerInterf
 
 func (l *l1SyncOrchestration) reset(startingBlockNumber uint64) {
 	log.Warnf("Reset L1 sync process to blockNumber %d", startingBlockNumber)
-	l.mutex.Lock()
-	defer l.mutex.Unlock()
-	comuserWasRunning := l.consumerStarted
-	if comuserWasRunning {
+	if l.isStarted {
```

Comment:
```suggestion
	if l.isRunning {
```
Just to use the same nomenclature than in line 53 and 54.

Reply:
good point, done